Movie_API/asgi.py

```diff
@@ -0,0 +1,16 @@
+"""

```

Comment:
Remane project directory, please. In Python, the snake case is preferable. Don't combine lowercase letters with uppercase ones.